demisto_sdk/commands/validate/tests/validators_test.py

```diff
@@ -635,6 +636,44 @@ def test_validate_no_missing_release_notes__missing_rn_in_modified_files(self, r
                                                                       old_format_files=set(),
                                                                       added_files=added_files) is False
 
+    def test_validate_no_missing_release_notes__missing_rn_dependent_on_api_module(self, repo, mocker, tmpdir):
```

Comment:
care to add another test which is positive(all packs depending on the apimodules have release notes as well)?